src/main/java/ladder/Points.java

```diff
@@ -0,0 +1,63 @@
+package ladder;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Points {
+    private static final Random random = new Random();
+
+    private List<Point> points = new ArrayList<>();
+
+    public Points(int count) {
+        for (int i = 0; i < count; i++) {
+            addPoint();
+        }
+    }
+
+    public Points(List<Point> points) {
+        this.points = points;
+    }
+
+    public List<Boolean> connections() {
+        return points.stream()
+                .map(Point::connected)
+                .collect(Collectors.toList());
+    }
+
+    private static boolean randomBoolean() {
+        return random.nextBoolean();
+    }
+
+    private void addPoint() {
+        points.add(new Point(connectable()));
+    }
+
+    private boolean connectable() {
+        if (points.isEmpty()) return randomBoolean();
+        if (points.stream().allMatch(point -> !point.connected())) return true;
```

Comment:
```suggestion
        if (points.stream().allMatch(point -> !point.connected())) {
            return true;
        }
```
마찬가지로 중괄호 넣어주세요. 추가로 if문의 조건은 별도 메서드로 빼서 이름을 지어주면 더 보기 좋을거 같아요.